css/amp.css

```diff
@@ -334,3 +334,13 @@ amp-analytics {
   overflow: hidden !important;
   visibility: hidden;
 }
+
+amp-user-notification {
+  position: fixed !important;
```

Comment:
@dvoytenko I think this is the problem

```css
.-amp-unresolved {
  position: relative !important;
  overflow: hidden !important;
}
```
https://github.com/ampproject/amphtml/blob/master/css/amp.css#L107

I think the correct fix is to remove that unresolved from that selector and instead just make it very specific. But possibly this is entirely wrong in the first place.